plugins/inputs/lustre2/lustre2.go

```diff
@@ -432,6 +455,90 @@ func (l *Lustre2) GetLustreProcStats(fileglob string, wantedFields []*mapping) e
 	return nil
 }
 
+func (l *Lustre2) GetLustreProcBrwStats(fileglob string, wantedFields []*mapping) error {
+	files, err := filepath.Glob(fileglob)
+	if err != nil {
+		return err
```

Comment:
You should prefix the error messages to ease later error tracing especially in functions that have multiple potential error exists.